solhint.js

```diff
@@ -113,6 +124,17 @@ function processPath(path) {
     return linter.processPath(path, readConfig());
 }
 
+
+/**
+     * Returns results that only contains errors.
+     * @param {LintResult[]} reporter The results to filter.
+     * @returns {LintResult[]} The filtered results.
+**/
+function getErrorResults(reporter) {
+    const reporterErrors = reporter[0].reports.filter(i => i.severity === 2);
```

Comment:
This is short and unlikely to be reused, I would do it inline.

Reply:
I agree, its a good observation.